### PR TITLE
Fixes #29522 - upgrade jquery-ui-rails

### PR DESCRIPTION
**Pull request by ezr-ondrej**
Upgrade dependend assets:
- jquery-ui-rails
- ~patternfly-react-sass~

~Packaging: theforeman/foreman-packaging#4379~
Packaging: theforeman/foreman-packaging#5030

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -1,8 +1,8 @@
-@import 'jquery.ui.theme';
-@import 'jquery.ui.core';
-@import 'jquery.ui.menu';
+@import 'jquery-ui/theme';
+@import 'jquery-ui/core';
+@import 'jquery-ui/menu';
 //we need to import all the above because we are using jquery-ui-rails and it uses sprockets require instaed of sass import.
-@import 'jquery.ui.spinner';
+@import 'jquery-ui/spinner';
```
Comment (mmoll) [inline app/assets/stylesheets/application.scss:5]: also needed in `app/assets/javascripts/application.js`

**Comment by mmoll** [comment]
Katello seems to need an update, too. :broken_heart:

**Comment by ezr-ondrej** [comment]
> Katello seems to need an update, too.

I've opened a Katello PR: Katello/katello#8450

**Comment by mmoll** [comment]
looks good, but needs RPM packaging

**Comment by mmoll** [comment]
@ezr-ondrej please rebase :)

**Comment by ezr-ondrej** [comment]
> @ezr-ondrej please rebase :)

The packaging still isn't ready. but sure, I can rebase now, to keep the track :)

**Comment by ezr-ondrej** [comment]
Blocked by: #7396

**Comment by mmoll** [comment]
@ezr-ondrej jquery-ui-rails alone could get split off into its own PR, correct?

**Comment by ezr-ondrej** [comment]
> @ezr-ondrej jquery-ui-rails alone could get split off into its own PR, correct?

Ja, that is correct! Thanks for pointing it out @mmoll :) Done.
Packaging: theforeman/foreman-packaging#5030

**Comment by evgeni** [comment]
[test katello]

**Comment by evgeni** [comment]
katello failures look "correct", they are like "couldn't find file 'jquery.ui.dialog'" which is what the attached Katello PR is fixing.

**Comment by ezr-ondrej** [comment]
only expected katello failures.

**Review by mmoll — APPROVED** [review]

**Comment by mmoll** [comment]
merged, díky @ezr-ondrej!

**Pull request merged**